### ISSUE TITLE
Is it a bug (increasing ram consumption indefinitely)

### ISSUE THREAD

**Issue by Sznymo** [simonkrauter/NiGui#175]
I noticed that when deleting button,etc....
ram consumption keeps increasing , so I ran a test (below its code and result).
After starting the code was eating 37.9 mb
After a few clicks 39.1 mb 
When there are many entries or buttons the ram consumption increases a lot . Is there a way to fix this ?

Code Test:
_import nigui
app.init()
var window = newWindow("Test")
window.width = 600.scaleToDpi
window.height = 400.scaleToDpi
var container = newLayoutContainer(Layout_Vertical)
window.add(container)
var button = newButton("Button Test Remove")
container.add(button)
button.onClick = proc(event: ClickEvent) =
    container.remove(button)
    container.add(button)
window.show()
app.run()_

![test](https://user-images.githubusercontent.com/116207604/235673098-ed60b29b-6eb8-443b-a748-a3750f1167e1.png)

**Comment by simonkrauter** [comment]
Are you running it under Linux/Gtk?
I have looked in the code of the ``remove`` and ``add`` methods, but I don't see anything which allocates memory.

**Comment by Sznymo** [comment]
Yes Linux/Gtk, this is how I add in the test button and remove it. 
Deleting does not release ram memory :/ there is no longer a button but still memory is taken. Similarly, deleting anything else in NiGui does not free up ram memory.

**Comment by simonkrauter** [comment]
Good news: I have found and fixed several memory leaks (https://github.com/simonkrauter/NiGui/commit/aa5908519eff9b9e80d0633be955f3744c64fef9).
At least with your example I don't see any more that the memory usage goes up.
If you still have such a problem, please re-open or create a new issue.
